sensor_stack/seabotix/src/control.py

```diff
@@ -37,34 +40,61 @@ def imuCB(dataIn):
 
 	prevError = errorP
 	errorP = goal - yaw
-	print errorP
+	fabserror=abs(errorP)
+	sign=1;
+	if(errorP<0):
+		sign=-1;
```

Comment:
@prudhvid @kalyan-kumar Are semi-colons really necessary? :stuck_out_tongue:  It is Python, after all.